shotgun_api3/shotgun.py

```diff
@@ -593,6 +599,17 @@ def find(self, entity_type, filters, fields=None, order=None,
         :param include_archived_projects: Optional, flag to include entities
         whose projects have been archived
 
+        :param additional_filter_presets: Optional list of presets to
+        further filter the result set, list has the form:
+        [ { preset_name: <preset_name>, <optional_preset parameters> } ],
+
+        Note that these filters are ANDed together and ANDed with the 'filter'
+        argument.
+
+        For details on supported presets and the format of this parameter,
+        please consult the API documentation:
+        https://github.com/shotgunsoftware/python-api/wiki
```

Comment:
When we do document this I believe it'll be at the following link, so we should probably be specific and not make somebody reading this hunt around:
https://github.com/shotgunsoftware/python-api/wiki/Reference%3A-Filter-Syntax